src/ManagementCenter.md

```diff
@@ -2,4 +2,4 @@
 
 Hazelcast Management Center enables you to monitor and manage your cluster members running Hazelcast. In addition to monitoring the overall state of your clusters, you can also analyze and browse your data structures in detail, update map configurations and take thread dumps from members. You can run scripts (JavaScript, Groovy, etc.) and commands on your members with its scripting and console modules.
 
-Please refer to [Hazelcast Management Center Documentation](http://docs.hazelcast.org/docs/management-center/3.8.3/manual/html/index.html) for its usage details.
+Please refer to [Hazelcast Management Center Documentation](http://docs.hazelcast.org/docs/management-center/3.9/manual/html/index.html) for its usage details.
```

Comment:
I think it will be better to use "latest". Could you please change this to http://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html ?

Reply:
Fixed.